### ISSUE TITLE
indicator.error is not valid  state role

### ISSUE THREAD

**Issue by o0shojo0o** [ioBroker/ioBroker.type-detector#23]
The role indicator.error is not valid.
This is used everywhere here!
https://github.com/ioBroker/ioBroker.docs/blob/master/docs/en/dev/stateroles.md#indicators-boolean-read-only

**Comment by Garfonso** [comment]
Looking at `indicator.maintenance - indicates system warnings/errors, alarms, service messages, battery empty or stuff like that` ERROR seems to be a bit redundant with MAINTENANCE. So one possibility would be to scrap ERROR and only use MAINTENANCE and allow the regex to accept `indicator.error`.

If ERROR should still be a different thing, then my proposal would be that the role should change to `indicator.maintenance.error` and docs and code be updated accordingly.

@GermanBluefox what is your opinion?

**Comment by Apollon77** [comment]
@GermanBluefox ? What do you think?

**Comment by Garfonso** [comment]
I can create a PR for it, if wanted.

**Comment by mcm1957** [comment]
I cannot see any error.xx roles here
https://github.com/ioBroker/ioBroker.docs/blob/master/docs/en/dev/stateroles.md#indicators-boolean-read-only

Possibly this Issue is obolete?

**Comment by Garfonso** [comment]
Yes, that is the problem. There are no indicator.error roles defined, but they are used in type-detector, for example here https://github.com/ioBroker/ioBroker.type-detector/blob/master/index.js#L80

**Comment by mcm1957** [comment]
OK, sorry - i read the issue the other way round. :-)

But I do not see if error.xxx is required. There are maintanance.xxx and alarm.xxx anyway.

**Comment by Garfonso** [comment]
Today I thought I'd create a PR, but I recognized that defaultType for ERROR is 'string'. So it seems to be meant for a error message or something? 

But that would not fit well with the `indicator.*` prefix, which is meant to be boolean. So, if ERROR should be meant to contain some human-readable error description, which might be useful (I have some devices that report something like that in KNX) to have a generic field for all possible devices, we might keep it but change the role to `text.error` and add the role to our ever-growing list of allowed roles. :-)

Any thoughts on this?
I could not find an Adapter that really uses/interprets the ERROR field, tbh. (Apart from lovelace Adapter, but lovelace interprets ERROR as boolean, currently, which is, as said above, redundant with MAINTAIN, so I'd change it there, too or get rid of it).

**Comment by Apollon77** [comment]
Was added on above page because missing there
